src/Testcontainers.Bigtable/BigtableBuilder.cs

```diff
@@ -0,0 +1,61 @@
+namespace Testcontainers.Bigtable;
+
+/// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+[PublicAPI]
+public class BigtableBuilder: ContainerBuilder<BigtableBuilder, BigtableContainer, BigtableConfiguration>
+{
+  public const string GCloudCliDockerImage = "gcr.io/google.com/cloudsdktool/google-cloud-cli";
```

Comment:
```suggestion
  public const string GCloudCliDockerImage = "gcr.io/google.com/cloudsdktool/google-cloud-cli:446.0.1-emulators";
```

Can we pin the image to this version? Then, we do not need to pull different versions and can reuse what the other modules are using.

Reply:
Agreed, I also updated documentation in the index under modules to reflect the change